HeterogeneousCore/AlpakaInterface/interface/host.h

```diff
@@ -0,0 +1,16 @@
+#ifndef HeterogeneousCore_AlpakaInterface_interface_host_h
+#define HeterogeneousCore_AlpakaInterface_interface_host_h
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+namespace alpaka_common {
+
+  // alpaka host device
+  static inline const DevHost host() {
```

Comment:
(sorry, missed this on my previous round) Should this be 
```suggestion
  static inline const DevHost& host() {
```
?

(this is my last comment, everything else looks good to me)

Reply:
Thanks for catching this, fixed.